src/store/utils/HistoryItem/utils/getTransactionFee.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+
+import { get } from 'lodash-es'
+import { getTxFee } from 'utils/transactions'
+
+export function getTransactionFee(
+  transaction: TransactionWithPrimaryKeys,
+  asset: DigitalAsset,
+): string {
+  const gasUsed = get(transaction, 'receiptData.gasUsed', 0)
+  const gasPrice = get(transaction, 'data.gasPrice', 0)
+  const decimals = get(asset, 'blockchainParams.decimals', 18)
+
+  return getTxFee(gasUsed, gasPrice, decimals)
+}
+
```

Comment:
btw, we already have similar util - https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/utils/transactions/getTxFee.js